docs/docs/reference/express/index.md

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
This file should be gitignored, as it's an autogenerated file. See https://github.com/nextauthjs/next-auth/blob/b20182c5fecf6997ad544a8069d2383bddfdcd2b/docs/docusaurus.config.js#L249-L263 for an example